dashboard/app/models/user_permission.rb

```diff
@@ -59,6 +59,11 @@ class UserPermission < ApplicationRecord
 
   after_save :log_permission_save
   before_destroy :log_permission_delete
+  after_create :send_verified_teacher_email, if: proc {permission == AUTHORIZED_TEACHER}
+
+  def send_verified_teacher_email
+    TeacherMailer.verified_teacher_email(user).deliver_now
```

Comment:
not sure if this is possible / remotely easy but can we add a test to verify this is triggered when adding an authorized teacher permission to a user (and maybe that it's not triggered when adding a different kind of permission)?

Reply:
i'll look into that - should be in the same area as the failing model test anyway

Reply:
finally got this in! https://github.com/code-dot-org/code-dot-org/pull/48265/files#diff-cbacbae99eb28182c4b023273a3d8a24aad329f124fb6a24329b11bff87d1502R213-R224